pyproject.toml

```
@@ -0,0 +1,35 @@
+[project]
+name = "sir"
+description="Search Index Rabbit"
+authors = [
+    { name = "MetaBrainz Foundation", email = "support@metabrainz.org" }
+]
```

Amending https://github.com/metabrainz/sir/pull/167/commits/80e08cb326769ec5fffc065280b99f00be543fdf#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L98 to pick up the original author name

```suggestion
authors = [
    { name = "MetaBrainz Foundation", email = "support@metabrainz.org" },
    { name = "Wieland Hoffmann" }
]
```

Didn’t pick up his email address as he is no longer active about it.